github3/repos/branch.py

```diff
@@ -440,16 +581,15 @@ def add_contexts(self, contexts):
         :rtype:
             list
         """
-        resp = self._post(self.contexts_url, json=contexts)
+        resp = self._post(self.contexts_url, data=contexts)
```

Comment:
Why did you switch to data here but use `json` below? Did I write something silly into `_post`?

Reply:
I've been testing variants of `json` vs `data`, figured out that when doing `_post` data is automatically encoded to JSON, for all other methods `data` is not encoded.